resources-ktx/build.gradle.kts

```diff
@@ -1,18 +1,19 @@
 import com.redmadrobot.build.dependencies.androidx
-import com.redmadrobot.build.dependencies.jetbrains
 
 plugins {
     id("com.redmadrobot.android-library")
     id("com.redmadrobot.publish")
-    id("kotlin-parcelize")
 }
 
 version = "1.3.1-0"
 description = "A set of Kotlin extensions for accessing resources"
 
+android {
+    namespace = "com.redmadrobot.extensions.resources-ktx"
+}
+
 dependencies {
-    api(jetbrains.kotlin.stdlib)
-    api(androidx.annotation)
+    api(projects.textvalue)
```

Comment:
@osipxd this dependency is not nice.

and even if we delete the Replace's, it will still remain because of `Fragment.getString(text: TextValue)`.
maybe make a textvalue-ktx module for views? 😔

Reply:
I think it's totaly OK